README.md

```diff
@@ -60,6 +66,8 @@ Then build just the ROS 1 bridge with `-j1`:
 src/ament/ament_tools/scripts/ament.py build --build-tests --symlink-install -j1 --only ros1_bridge
 ```
 
+*Note:* If you have built your ROS 2 workspace using `--isolated`, you will need to add the message/service packages you want to bridge to your path before building the bridge, e.g. by sourcing `local_setup.*`.
```

Comment:
the packaging job (non-isolated) doesn't seem to do it. I understood that in non-isolated builds the packages installed are on the path by default?